networking/kvm.go

```diff
@@ -104,7 +105,7 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf) (netlink.Link,
 	if config.MTU != 0 {
 		mtu = config.MTU
 	}
-	nameTemplate := fmt.Sprintf("rkt-%s-vtap%%d", podID.String()[0:4])
+	nameTemplate := fmt.Sprintf("rkt-%s-vtap%d", podID.String()[0:4], interface_number)
```

Comment:
You are right - the idea was to push template to kernel side (what is working with other network drivers, but not with macvtap) and was modified to prepare this name directly on rkt side.
So, I should update variable name.